crates/ra_hir/src/code_model_impl/module.rs

```diff
@@ -143,6 +143,10 @@ impl Module {
                         return PerNs::none();
                     }
                 }
+                PathKind::Abs => {
```

Comment:
I’ll write an issue for this later, but the idea f the fix would be to add an “extern_crates_scope” method to the Crate struct, which returns a mapping from names to DefIds of the root module, and use it here, in resolve import and in polulate_module